rules/limit.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
     rules: {
         // Color
         'color-named': 'never',
-        'color-no-hex': true,
+        // 'color-no-hex': true,
```

Comment:
Why are we keeping these as comments?

Reply:
To be consistent with the rest of the file. Once we start relying on CSS linting a bit more, we'll probably want to experiment with turning rules on and off a bit.